GVRf/Framework/jni/engine/importer/assimp_importer.cpp

```diff
@@ -68,4 +71,176 @@ std::shared_ptr<Mesh> AssimpImporter::getMesh(int index) {
 
     return std::shared_ptr < Mesh > (mesh);
 }
+
+void AssimpImporter::sceneRecursion(aiNode* node, const aiScene* aScene, std::shared_ptr<Scene> scnPtr, JNIEnv * env, jobject obj, jobject bitmap, aiMatrix4x4 accumulatedTransform)
+{
```

Comment:
We have been careful to use consistent (automatic) formatting throughout GVRF, so that reviewers can focus on 'real' changes, not whitespace changes.

In Eclipse, we use very slightly modified versions of the "K & R" built-in (for C++) and the "Java Conventions" built-in (for, obviously, Java). In both cases, the only change we've made is to set the "Tab policy" to "Spaces only" (from the default "Mixed".)

I will keep on reviewing, but will not merge this checkin until you've applied the standard formatting and updated the patch.